### PR TITLE
feat: added homepage

### PR DESCRIPTION
**Pull request by kshitijthatte**
added the homepage, here categories are being fetched from the backend

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
```
Comment (msdeshmukh009) [inline public/index.html]: You can remove these comments now.

File: src/pages/Homepage/styles.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+.category-img {
+  height: 100%;
+  border-radius: 4px;
+}
+
+.best-selling .card,
+.reviews .card {
+  justify-self: center;
+  /* min-width: 30vw; */
```
Comment (msdeshmukh009) [inline src/pages/Homepage/styles.css]: You can remove this comment if you don't need it.

**Review by manojasarsa — APPROVED** [review]
The code looks clean. Good to merge!

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking good. Good to merge.

**Review by tanveertkd — APPROVED** [review]
This is good to go!

**Pull request merged**